static/app/views/settings/account/apiTokens.tsx

```diff
@@ -7,127 +7,149 @@ import AlertLink from 'sentry/components/alertLink';
-          )}
-        </AlertLink>
-        <TextBlock>
-          {t(
-            "Authentication tokens allow you to perform actions against the Sentry API on behalf of your account. They're the easiest way to get started using the API."
-          )}
-        </TextBlock>
-        <TextBlock>
-          {tct(
-            'For more information on how to use the web API, see our [link:documentation].',
-            {
-              link: <ExternalLink href="https://docs.sentry.io/api/" />,
-            }
+  const isEmpty = !Array.isArray(tokenList) || tokenList.length === 0;
+
+  const action = (
+    <Button
+      priority="primary"
+      size="sm"
+      to="/settings/account/api/auth-tokens/new-token/"
+      data-test-id="create-token"
+    >
+      {t('Create New Token')}
+    </Button>
+  );
+
+  return (
+    <SentryDocumentTitle title={PAGE_TITLE}>
+      <SettingsPageHeader title={PAGE_TITLE} action={action} />
+      <AlertLink to={`/settings/${organization?.slug ?? ''}/auth-tokens/`}>
+        {t(
+          "User Auth Tokens are tied to the logged in user, meaning they'll stop working if the user leaves the organization! We suggest using Organization Auth Tokens to create/manage tokens tied to the organization instead."
+        )}
+      </AlertLink>
+      <TextBlock>
+        {t(
+          "Authentication tokens allow you to perform actions against the Sentry API on behalf of your account. They're the easiest way to get started using the API."
+        )}
+      </TextBlock>
+      <TextBlock>
+        {tct(
+          'For more information on how to use the web API, see our [link:documentation].',
+          {
+            link: <ExternalLink href="https://docs.sentry.io/api/" />,
+          }
+        )}
+      </TextBlock>
+      <Panel>
+        <PanelHeader>{t('Auth Token')}</PanelHeader>
+
+        <PanelBody>
+          {isEmpty && (
+            <EmptyMessage>
+              {t("You haven't created any authentication tokens yet.")}
+            </EmptyMessage>
           )}
-        </TextBlock>
-        <Panel>
-          <PanelHeader>{t('Auth Token')}</PanelHeader>
-
-          <PanelBody>
-            {isEmpty && (
-              <EmptyMessage>
-                {t("You haven't created any authentication tokens yet.")}
-              </EmptyMessage>
-            )}
-
-            {tokenList?.map(token => (
-              <ApiTokenRow
-                key={token.id}
-                token={token}
-                onRemove={this.handleRemoveToken}
-              />
-            ))}
-          </PanelBody>
-        </Panel>
-      </div>
-    );
-  }
+
+          {tokenList?.map(token => (
+            <ApiTokenRow key={token.id} token={token} onRemove={deleteToken} />
+          ))}
+        </PanelBody>
+      </Panel>
+    </SentryDocumentTitle>
+  );
 }
 
 export default withOrganization(ApiTokens);
```

Comment:
Looks like you can remove `withOrganization` now 🙌

Reply:
Good catch! 👍